src/pymovements/dataset/dataset_definition.py

```diff
@@ -132,9 +141,9 @@ class DatasetDefinition:
     name: str = '.'
     has_files: dict[str, bool] = field(default_factory=dict)
 
-    mirrors: dict[str, tuple[str, ...]] = field(default_factory=dict)
+    mirrors: dict[str, list[str]] = field(default_factory=dict)
```

Comment:
Isn't this an unnecessary breaking change or is the type cast implicit if users have defined their own `DatasetDefinition`?

Couldn't we just use `mirrors: dict[str, list[str] | tuple[str, ...]]` instead for backwards compatibility?

The actual dataset definitions for the public datasets (e.g. in `bsc.py`) will probably need `list` to pass the equality tests to the yamls right?

Reply:
changed it